.github/workflows/build_deploy.yml

```diff
@@ -4,18 +4,18 @@ on:
   push:
     branches:
       - "6-dev"
+      - "update_workflows"
```

Comment:
Should we use this branch for updating workflows in the future, or is it just a one-time thing and should be removed before merging?

Reply:
this is just temporarily because otherwise the worklow doesn't run and the first real test will be when it gets merged to main/master.  Like what happened with our updated set variable line.